README.rst

```diff
@@ -355,6 +355,13 @@ To build your application locally (for local testing), say this in any directory
 
     cartridge build [PATH] [flags]
 
+The following options (``[flags]``) are supported:
+
+* ``--spec`` is the path to rockspec to use for current build.
+
+**Note**: Please use rockspec name in lower case when you build
+your application with ``--spec`` flag.
+
```

Comment:
Maybe also add this information to ``--help`` command? Something like this:

```
Path to rockspec to use for current build.
Should be in lowercase.
```

And remove **Note** from the docs, but add into description:

```
* ``--spec`` is the path to rockspec to use for current build. Should be in lowercase.
```

Reply:
About `--help` flag: this flag is supported by all commands but there is no description for this flag for any command in doc. My point is if we want to add such information this should be applied to description for each command and looks like it is task for another patch.

Reply:
Agree. We need a separate patch.